perf/security/mtls/setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#set -ex
+
+WD=$(dirname "$0")
+WD=$(cd "${WD}" || exit; pwd)
+
+function setup_test() {
+  local NAMESPACE="mtls"
+  local HELM_FLAGS=${HELM_FLAGS:-""}
+
+  mkdir -p "${WD}/tmp"
+  local OUTFILE="${WD}/tmp/${NAMESPACE}.yaml"
+
+  kubectl create ns "${NAMESPACE}" || true
+  kubectl label namespace "${NAMESPACE}" istio-injection=enabled || true
+
+  helm --namespace "${NAMESPACE}" -f ./values.yaml --set "${HELM_FLAGS}" \
+    template "${WD}" > "${OUTFILE}"
+
+  if [[ -z "${DRY_RUN}" ]]; then
+      kubectl --namespace "${NAMESPACE}" apply -f "${OUTFILE}"
+      pushd ../../load/loadclient || exit
+      # Generate larger load for load client.
+      export LOADCLIENT_EXTRA_HELM_FLAGS="--set qps=200"
+      ./setup_test.sh "${NAMESPACE}" "svc-"
+      popd || exit
+  fi
+}
+
+function setup_istio() {
+  pushd ../../istio-install || exit
+  export OPERATOR_PROFILE="automtls.yaml" OPERATOR_SHA='438827b1602037fe30dedcd0008ce0cae0ef0aee' &&
```

Comment:
can you update the operator sha file instead

Reply:
This SHA is grabbed from 1.4 HEAD.

If I remembered correctly, without being specific in the test, the HEAD sha or the sha file is not working for my test suite.

Maybe I'll remove this one and add doc in README instead.

I've experienced several times of test suite is broken after a while, due to the dependency changes, (load client installation), or the Istio installation slight changes.

I think the solution should be having each test suite defines some validation hook, and setup some CI to verify the installation meets the test suites' needs.

```shell
source setup.sh && validate
```

Reply:
moved to instructions instead.